runner/execer/os/os_execer.go

```diff
@@ -62,6 +62,7 @@ func (e *osExecer) Exec(command execer.Command) (result execer.Process, err erro
 		cmd.Stderr = stderrW.WriterDelegate()
 	}
 
+	// Sets pgid of all child processes to cmd's pid
```

Comment:
Nope, just pgid. there's a Setsid: true option too, but I think it's better we don't touch that. That way we can more accurately measure session memory usage. This is from inside a worker in aurora for reference.
![screen shot 2017-05-10 at 11 28 53 am](https://cloud.githubusercontent.com/assets/3813401/25907035/4a821d38-3574-11e7-80f7-b8f6d97d9d98.png)